### PR TITLE
Add run:cleanup command

### PR DESCRIPTION
**Pull request by rafaelgomesxyz**
Sometimes run workloads are not deleted and keep hanging, despite having an `ensure` to delete them. This command is useful for cleaning up any stale run workloads when that happens.

Example output:

Code_KKUSAscquk

### CODE REVIEW

File: docs/commands.md [hunk 1]
```diff
@@ -278,6 +278,17 @@ cpl run bash -a $APP_NAME -w other-workload
 cpl run bash -a $APP_NAME --use-local-token
 ```
 
+### `run:cleanup`
+
+- Deletes stale run workloads for an app
+- Stale run workloads are identified based on the creation date of the workload
+- Specify the amount of days after a run workload should be considered stale through `stale_run_workload_created_days` in the `.controlplane/controlplane.yml` file
+- Will ask for explicit user confirmation
```
Comment (justin808) [inline docs/commands.md:286]: is this what you mean?
> Will ask for explicit user confirmation of deletion.

File: lib/command/run_cleanup.rb [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    NAME = "run:cleanup"
+    OPTIONS = [
+      app_option(required: true),
+      skip_confirm_option
+    ].freeze
+    DESCRIPTION = "Deletes stale run workloads for an app"
+    LONG_DESCRIPTION = <<~DESC
+      - Deletes stale run workloads for an app
+      - Stale run workloads are identified based on the creation date of the workload
+      - Specify the amount of days after a run workload should be considered stale through `stale_run_workload_created_days` in the `.controlplane/controlplane.yml` file
```
Comment (justin808) [inline lib/command/run_cleanup.rb:14]: see above comment regarding wording of this param

File: lib/command/run_cleanup.rb [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            next unless diff_in_days >= stale_run_workload_created_days
+
+            run_workloads.push({
+                                 name: workload_name,
+                                 date: created_date
+                               })
+          end
+
+          run_workloads
+        end
```
Comment (justin808) [inline lib/command/run_cleanup.rb:82]: can we have specs for this method?

File: lib/command/run_cleanup.rb [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+      - Specify the amount of days after a run workload should be considered stale through `stale_run_workload_created_days` in the `.controlplane/controlplane.yml` file
+      - Will ask for explicit user confirmation
+    DESC
+
+    def call
+      return progress.puts("No stale run workloads found.") if stale_run_workloads.empty?
+
+      progress.puts("Stale run workloads:")
+      stale_run_workloads.each do |workload|
+        progress.puts("  #{workload[:name]} (#{Shell.color((workload[:date]).to_s, :red)})")
```
Comment (justin808) [inline lib/command/run_cleanup.rb:23]: Can you list how many days ago?

File: lib/command/run_cleanup.rb [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+          workloads = cp.workload_query("-run-", partial_match: true)["items"]
+          workloads.each do |workload|
+            workload_name = workload["name"]
+
+            original_workload_name, workload_number = workload_name.split("-run-")
+            next unless defined_workloads.include?(original_workload_name) && workload_number.match?(/^\d{4}$/)
+
+            created_date = DateTime.parse(workload["created"])
+            diff_in_days = (now - created_date).to_i
```
Comment (justin808) [inline lib/command/run_cleanup.rb:72]: Are you 100% Ruby returns the difference in date **time** in terms of **days**?

File: lib/command/run_cleanup.rb [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+            next unless diff_in_days >= stale_run_workload_created_days
```
Comment (justin808) [inline lib/command/run_cleanup.rb:73]: Do we need to worry about any rounding errors?

**Review by justin808 — CHANGES_REQUESTED** [review]
See comments

File: lib/command/run_cleanup.rb (hunk 3 above)
Comment (rafaelgomesxyz) [reply lib/command/run_cleanup.rb:84]: Done.

**Review by justin808 — APPROVED** [review]

**Review by justin808 — APPROVED** [review]

**Pull request merged**